### PR TITLE
Integration core with UI and basic commands

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

File: static/js/ScreenManager.js [hunk 1]
```diff
@@ -8,7 +11,13 @@ class ScreenManager {
...
         this.terminalSections = {};
         this.terminalButtons = {};
-
+        this.commandLine = null;
+        this.commandLineButton = null;
+        window.logoInterpreter.handleCreateLine((event, value) => {
+            console.log(value);
+            CanvasManager.getInstance().addDrawableObject(new DrawableLine(value.x, value.y, value.x2, value.y2, value.width, value.color));
+        });
+        // window.logoInterpreter.execute("TO func :x :y CS END TO run TO func2 :aaa CS END CS HT ST PU PD RT 10 LT 10 FD 10 BK 10 SETC 10 SETPN 2 SETPC 1 12 POTS ERALL END run");
```
Comment (jaanonim) [inline static/js/ScreenManager.js]: ?

File: core/models/turtle.js [hunk 2]
```diff
@@ -1,49 +1,84 @@
...
     copy(id) {
         return new Turtle(id, this.x, this.y, this.rotation);
     }
 
     forward(distance) {
-        console.log(this.id + " forward " + distance);
+        let interpreterObject = this.Global.getInterpreterObjects();
+        let newX = this.x + distance * Math.cos(this.degree2rad(this.rotation - 90));
+        let newY = this.y + distance * Math.sin(this.degree2rad(this.rotation - 90));
+        // console.log(this.rotation, this.degree2rad(180 + this.rotation), "|", this.x, this.y, "|", newX, newY);
```
Comment (jaanonim) [inline core/models/turtle.js]: ?

File: static/js/drawableLine.js [hunk 3]
```diff
@@ -17,10 +17,8 @@ class DrawableLine {
         ctx.beginPath();
         ctx.moveTo(x, y);
         ctx.lineTo(x2, y2);
-        ctx.strokeStyle = "red";
-        ctx.lineWidth = 1;
-        // ctx.strokeStyle = this.color;
-        // ctx.lineWidth = this.width;
```
Comment (jaanonim) [inline static/js/drawableLine.js:23]: ?

File: testingCommands.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+FD 100 RT 90 FD 100 RT 90 FD 100 RT 90 FD 100
+REPEAT 360 [lt 1 fd 1]
+REPEAT 36 [REPEAT 360 [lt 1 fd 1] RT 10]
+FD 100 PU FD 100 PD FD 100
+SETC 1 SETPN 1 SETPC 1 1
```
Comment (jaanonim) [inline testingCommands.txt:5]: This should be in git?

**Review by jaanonim — APPROVED** [review]
LGTM

File: testingCommands.txt (hunk 4 above)
Comment (n2oneProgrammer) [reply testingCommands.txt:5]: Yes, for ginder head and another person in team

**Pull request merged**